tests/unit/brayns/engine/TestSimulationScanner.cpp

```diff
@@ -0,0 +1,106 @@
+/* Copyright (c) 2015-2022, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible author: Nadir Roman Guerrero <nadir.romanguerrero@epfl.ch>
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include <brayns/engine/common/SimulationScanner.h>
+#include <brayns/engine/components/SimulationInfo.h>
+#include <brayns/engine/core/Engine.h>
+#include <brayns/engine/scene/ModelManager.h>
+
+#include <doctest/doctest.h>
+
+namespace
+{
+class AddSimulatedModels
+{
+public:
+    static void add(brayns::ModelManager &models, double start, double end, double dt, bool enabled)
+    {
+        auto model = std::make_shared<brayns::Model>("test");
+        auto &components = model->getComponents();
+        auto &simInfo = components.add<brayns::SimulationInfo>();
+        simInfo.dt = dt;
+        simInfo.enabled = enabled;
+        simInfo.endTime = end;
+        simInfo.startTime = start;
+        models.add(std::move(model));
+    }
+};
+}
+
+TEST_CASE("simulation_scanner")
+{
+    auto params = brayns::ParametersManager(0, nullptr);
+    auto engine = brayns::Engine(params);
+    (void)engine;
```

Comment:
PlaceholderEngine ?

Reply:
It initializes the ospray device, needed when commiting objects